.github/workflows/node.js.yml

```
@@ -1,5 +1,9 @@
 name: Node.js CI
 
+strategy:
```

actionlint (1.7.8)

unexpected key "strategy" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)